src/main/kotlin/no/nav/familie/ef/mottak/hendelse/JournalpostUtil.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.familie.ef.mottak.hendelse
+
+import io.micrometer.core.instrument.Metrics
+import no.nav.familie.kontrakter.felles.journalpost.Journalpost
+import no.nav.familie.kontrakter.felles.journalpost.Journalstatus
+
+
+fun Journalpost.skalBehandles() = this.erTemaEnfOgTypeI() && this.journalstatus == Journalstatus.MOTTATT && this.gyldigKanal()
```

Comment:
Siden dette er spesifikke behov for Enslig Forsørger og ikke likt behovene til BAKS tenker jeg at det er best å ha en util-metode som utvider Journalpost-objektet her i `familie-ef-mottak` og IKKE i `familie-kontrakter` 🎉